src/os/layer/tile.js

```diff
@@ -364,6 +471,7 @@ os.layer.Tile.prototype.applyColors = function(data) {
         // transformColor blends between the src and target color, leaving densitization intact
         os.color.transformColor(data, srcColor, tgtColor);
```

Comment:
We should skip transforming the color if the layer is using the default color. Changing brightness/contrast/saturation without changing the layer color should adjust those values against the original tiles. Currently it re-colors the tiles white, then makes the adjustment.